dataeng/resources/vertica-replica-import.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
```

Comment:
Same thing, use `-z`.

Reply:
Actually, why is this here?   Either we should not calculate this at all, or we should use it in the call to run-automated-task below (which we're not currently doing).  Then again, if NUM_TASK_INSTANCES here is zero, then there's little point in specifying 0 for --num-reduce-tasks (or whatever it is).